src/pages/ShareMessagePage.js

```diff
@@ -9,27 +9,27 @@ import ScreenWrapper from '../components/ScreenWrapper';
 import TextInput from '../components/TextInput';
 import withLocalize, {withLocalizePropTypes} from '../components/withLocalize';
 // import additionalAppSetup from './src/setup';
-import OptionRowLHN from '../components/LHNOptionsList/OptionRowLHN';
+import CONST from '../CONST';
+import OptionRowLHNData from '../components/LHNOptionsList/OptionRowLHNData';
 import Navigation from '../libs/Navigation/Navigation';
 import * as Report from '../libs/actions/Report';
 import styles from '../styles/styles';
 
 function ShareMessagePage(props) {
     const reportID = props.route.params.reportID;
-    const [attachment, setAttachment] = useState(props.route.params.share);
-    const [message, setMessage] = useState('');
+    const attachment = props.route.params.share;
+    const source = attachment.data;
+    const uri = attachment.data;
+    const name = source.split('/').pop();
```

Comment:
note, also for polish, would be good to have utility functions in lib for share data